docs/css/front-page.css

```
@@ -98,48 +87,240 @@ h5.tile-title {
+    margin-left: 0;
+    text-align: center;
+    flex-direction: column;
+    justify-content: center;
+}
+
+[dir=ltr] .info-tile-body ul li {
+    list-style-type: none;
+    background-size: 15px 18px;
+    text-align: center;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 0px;
+}
+
+.info-tile-title {
+    color: #ffffff;
+    font-family: "Noto Sans";
     font-size: 18px;
-    color: var(--ibexa-jazzberry);
+    font-weight: bold;
+    line-height: 50px;
+    padding-left: 0;
+    text-align: center;
 }
 
-.announcement a.external:after {
-    content: none;
+h5.info-tile-title {
+    color: #ffffff;
+    font-family: "Noto Sans";
+    font-size: 18px;
+    font-weight: bold;
+    line-height: 50px;
+    padding-left: 0;
+    text-align: center;
 }
 
-.announcement .btn {
-    font-size: 0.8rem;
-    padding: .4rem .8rem;
+.info-tile-icon {
+    margin-right: 0px;
+    margin-top: 0px;
+    margin-left: 0px;
+    margin-bottom: 0px;
 }
 
-.announcement .tile-icon {
-    flex-shrink: 0;
+  .info-tile-up:hover {
+    opacity:0.5;
+  }
+
+a.instruction-tile {
+    display: inline-flex;
+    align-items: center;
+    height: 150px;
+    width: 24%;
+    margin: 0.5%;
+    max-height: 90px;
+    border-radius: 5px;
+    padding-left: 1vw;
+    padding-right: 1vw;
+    background: #af1164;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    word-wrap: break-word;
+    font-family: "Noto Sans";
+    color: white;
+    font-weight: bold;
+    position: relative;
+    margin-bottom: 8px;
+    text-align: center;
+    margin-top: 8px;
+    flex-direction: column;
+    justify-content: center;
+    color: white;
 }
 
-.announcement .btn-announcement {
-    background: var(--ibexa-jazzberry);
-    border: 1px solid var(--ibexa-jazzberry);
-    color: #FFFFFF;
+a.instruction-tile:hover {
+  opacity:0.5;
+  color: white;
 }
 
-.announcement .btn-announcement-outline {
-    background: #FFFFFF;
-    border: 1px solid var(--ibexa-jazzberry);
-    color: var(--ibexa-jazzberry);
+html {
+    height: 100%;
+  }
+  body {
+    margin: 0;
+    /* background: linear-gradient(231deg, #ffffff 0%, #f0f0f0 99%); */
+    padding-top: 100px;
+    font-family: "Noto Sans";
+  }
+  
+  .accordion {
+    margin: 0;
+    width: 100%;
+    background-color: #fff;
+    padding: 15px;
+    border-radius: 10px;
+    box-shadow: 0px 3px 15px rgba(0, 0, 0, 0.2);
+    opacity: 0.8;
+    font-size: 14px;
+  }
+ 
+  .accordion details summary {
+    line-height: 50px;
+    padding-left: 15px;
+    padding-right: 15px;
+    font-weight: 400;
+    cursor: pointer;
+    border-radius: 10px;
+    position: relative;
+    user-select: none;
+  }
+ 
+  .accordion details[open] summary,
+  .accordion details summary:hover {
+    background-color: rgba(0, 0, 0, 0.05);
+  }
+  .accordion details[open] summary{
+    font-weight: bold;
+  }
+  
+  .accordion details ::marker {
+    font-size: 0;
+  }
+  .accordion details ul {
```

spacing - here and in other places
```suggestion
  }
  
  .accordion details ul {
```